ql/src/test/results/clientpositive/llap/groupby1_limit.q.out

```diff
@@ -71,33 +71,34 @@ STAGE PLANS:
                 mode: mergepartial
                 outputColumnNames: _col0, _col1
                 Statistics: Num rows: 316 Data size: 30020 Basic stats: COMPLETE Column stats: COMPLETE
-                Limit
-                  Number of rows: 5
-                  Statistics: Num rows: 5 Data size: 475 Basic stats: COMPLETE Column stats: COMPLETE
-                  Reduce Output Operator
-                    null sort order: 
-                    sort order: 
-                    Statistics: Num rows: 5 Data size: 475 Basic stats: COMPLETE Column stats: COMPLETE
-                    TopN Hash Memory Usage: 0.1
-                    value expressions: _col0 (type: string), _col1 (type: double)
+                Reduce Output Operator
```

Comment:
we lost the `Limit` operator from here - as a result we will be shuffling all input rows.
I think this could become more costly for larger tables than the old plan.

I don't see TopN hash enabled on the reduce operator - which could possibly save the day in this case; why did we loose that as well?

Reply:
But we still have TopNKey operator in the Mapper (both old and new plan) it filters out the majority of the rows.

This query has the same issue like the example in the jira: it has gby with limit + aggregate function in the project:
```
SELECT src.key, sum(substr(src.value,5)) GROUP BY src.key LIMIT 5
``` 
If no ordering is specified we may end up with incorrect aggregations.

Reply:
I missed that - most likely because the the row estimate was >100.
In that case this doesn't seem to be a problem; however we should fix the stat estimate for the TNKO - could you open a ticket?